scipy/cluster/hierarchy.py

```diff
@@ -1151,7 +1177,7 @@ def cophenet(Z, Y=None):
     Zs = Z.shape
     n = Zs[0] + 1
 
-    zz = np.zeros((n * (n-1)) // 2, dtype=np.double)
+    zz = np.zeros((n * (n - 1)) // 2, dtype=np.double)
```

Comment:
can you revert this change too? the old version is more readable and fine for PEP8